AllReadyApp/Web-App/AllReady/Startup.cs

```diff
@@ -169,7 +169,7 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
 
             //Hangfire
             services.AddHangfire(configuration => configuration.UseSqlServerStorage(Configuration["Data:HangfireConnection:ConnectionString"]));
-
+            
             services.AddScoped<IAllReadyUserManager, AllReadyUserManager>();

```

Comment:
This file seems to only have whitespace changes - perhaps it could be excluded/rolled-back from the PR (i.e. `github checkout origin/master -- AllReadyApp/Web-App/AllReady/Startup.cs`)